chainer/backends/intel64.py

```diff
@@ -87,17 +87,24 @@ def should_use_ideep(level):
 
 
 def inputs_all_ready(inputs, supported_ndim=(2, 4)):
-    """Checks if input arrays are supported for ideep optimization.
+    """Checks if input arrays are supported for the iDeep primitive.
 
+    Before calling iDeep primitives (e.g., ``ideep4py.linear.Forward``), you
+    need to make sure that all input arrays are ready for the primitive by
```

Comment:
"iDeep primitives" -> "an iDeep primitive", OR "the primitive" -> "the primitives"